python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -306,7 +315,13 @@ def poll_sync(
                 curr_last_sync_succeeded,
                 curr_sync_state,
             ) = self.get_connector_sync_status(connector_id)
+
             self._log.info(f"Polled '{connector_id}'. Status: [{curr_sync_state}]")
+            if self.error_on_reschedule and curr_sync_state == "rescheduled":
+                raise Failure(
```

Comment:
maybe raise a named exception here since this is more of an exceptional condition then an explicit "failure"